andyMe/Main.jsx

```diff
@@ -1,102 +1,90 @@
-        <Text style={estilasos.primerTexto}>Mi comida favorita:<Text style={estilasos.enef}> Espagueti a la boloñesa </Text><TouchableOpacity onPress={()=> Alert.alert('El espagueti a la boloñesa es una comida tipica de Italia ☝️🤓')}><Text style={estilasos.cuestion}>?</Text></TouchableOpacity></Text>
-        <Text style={estilasos.primerTexto}>Mi pelicula favorita:<Text style={estilasos.enef}> Snoopy: la pelicula </Text><TouchableOpacity onPress={()=> Alert.alert('No me da pena admitirlo, es la mejor pelicula')}><Text style={estilasos.cuestion}>?</Text></TouchableOpacity></Text>
-        <Text style={estilasos.primerTexto}>Mi videojuego favorito:<Text style={estilasos.enef}> Titanfall 2 </Text><TouchableOpacity onPress={()=> Alert.alert('Titanfall 2 es un juego que salio en 2016 y es una obra maestra.')}><Text style={estilasos.cuestion}>?</Text></TouchableOpacity></Text>
-        <Text style={estilasos.primerTexto}>Budget: $22 (pesos)<Text style={estilasos.enef}> </Text></Text>
-    </View>
-    <Text></Text>
-    <View style={[estilasos.quintoView, { backgroundColor: isEnabled ? '#1d8e5a' : '#e7d667' }]}>
-        <Text style={estilasos.numberText}>5.- Programación 🖥️🤓</Text>
-        <Text style={estilasos.titulos}>Razones</Text>
-        <Text style={estilasos.chiquillos}>Empecé en este mundo hace bastante poco y fue para ayudar a un tío en una tarea. Aunque sinceramennte me enamoré del código y me gusta bastante programar, aunque no sea muy bueno. Me gustaría mencionar que quien me inspiró para continuar fue mi primo Santi, de hecho gracias a él estoy en React native!</Text>
-        <Text style={estilasos.titulos}>Hasta ahora</Text>
-        <Text style={estilasos.chiquillos}>Al inicio, fue con el lenguaje c++ aunque no le entedí ni un poco. Entonces empecé un curso de html y css, en esos no era tan malo. Luego, justamente mi primo me dijo que aprendiera Javascript, aunque no soy muy bueno en el lenguaje, continué. Y como se ve ahora, estoy con las dos biblitecas de JS, React y React Native. </Text>
-        <Text style={estilasos.titulos}>Futuro</Text>
-        <Text style={estilasos.chiquillos}>Me gustaría volverme un programador full-stack principalmente de esta biblioteca. Aprender una par de lenguajes más y practicar los que "sé" actualmente. También me gustaría poder trabajar en esto. Muchas gracias por llegar hasta aquí.</Text>
-        
-        <Text></Text>
-    </View>
-    <Text></Text>
-    <Text></Text>
-    <Text></Text>
-    <Text></Text>
-    <Text></Text>
-    </View>
-    <View style={estilasos.final}>
-        <Text></Text>
-        <Text style={estilasos.finalText}>Muchas gracias por leer. Como lo pueden notar, no es el proyecto más bonito, pero quería probar todos mis conocimientos actuales de React Native, luego de casi un mes. Ya estoy trabajando en la siguiente aplicación, estará lista para el 25 de abril. ¡Gracias!</Text>
-        <Text></Text>
-    </View>
-    </ScrollView>
+    <SafeAreaView>    
```

Comment:
Check out this UI when I open your app in a notched phone:

![Simulator Screenshot - iPhone 15 Pro Max - 2024-04-13 at 15 39 25](https://github.com/Andy-Erick/My-first-app/assets/56652644/145d0ac0-2a4d-48c9-bef8-5f814375c119)

With safe area view we fix that.

![Simulator Screenshot - iPhone 15 Pro Max - 2024-04-13 at 15 39 32](https://github.com/Andy-Erick/My-first-app/assets/56652644/ae5adffd-8477-49ef-99c9-0d371e174f44)